libs/shared/src/types/features.ts

```diff
@@ -20,6 +20,7 @@ export enum Feature {
   PromptsPublishing = 'prompts-publishing',
   CustomLogo = 'custom-logo', // Enable setting for custom logo feature
   HideNewConversation = 'hide-new-conversation', // hide "New conversation" button
+  HideCustomApplications = 'hide-custom-applications', // hide custom applications
```

Comment:
We need opposite feature: `CustomApplications` to enable instead of `HideCustomApplications` to disable

Reply:
done